docs/src/content/docs/commands/JSON.CLEAR.md

```diff
@@ -5,114 +5,91 @@ description: Documentation for the DiceDB command JSON.CLEAR
 
 The `JSON.CLEAR` command is part of the DiceDBJSON module, which allows you to manipulate JSON data stored in DiceDB. This command is used to clear the value at a specified path in a JSON document, effectively setting it to an empty state. This can be particularly useful when you want to reset a part of your JSON document without removing the key itself.
```

Comment:
Hi, as per recent discussion on DiceDb community, DiceDB supports JSON natively hence its not a part of DiceDBJSON module. Would it be okay to update this as well in the description.

Reply:
Hey @apoorvyadav1111, changed to remove `part of the DiceDBJSON module`